File: PYTHON PROGRAMMING/Test63.py
# 63. PALINDROME NUMBER
from math import *
def Palindrome(num):
    NUM = num
    rev = 0
    while num != 0:
        rem = num % 10
        rev = (rev * 10) + rem
        num = floor(num / 10)
    if NUM == rev:
        print(f"{NUM} Is A Palindrome Number.")
    else:
        print(f"{NUM} Is Not A Palindrome Number.")
Palindrome(121)
Palindrome(153)

File: TYPESCRIPT PROGRAMMING/Test63.ts
// 63. STRONG NUMBER
const StrongNumber = (num: number):void => {
    let temp: number = 1, NUM: number = num, count: number = 0;
    while(num !== 0) {
        let rem: number = num % 10;
        for(let i = 1; i <= rem; i++) {
            temp *= i;
        }
        count += temp;
        temp = 1;
        num = Math.floor(num / 10);
    }
    if(NUM === count) {
        console.log(`${NUM} Is A Strong Number.`);
    } else {
        console.log(`${NUM} Is Not A Strong Number.`);
    }
}
StrongNumber(145);
StrongNumber(265);